Engines/Wine/Verbs/VK9/script.js

```diff
@@ -0,0 +1,59 @@
+include(["engines", "wine", "engine", "object"]);
+include(["engines", "wine", "plugins", "override_dll"]);
+include(["utils", "functions", "net", "resource"]);
+include(["utils", "functions", "filesystem", "files"]);
+
+/**
+* Setup VK9-> https://github.com/disks86/VK9
+* @param {String} vk9Version VK9 version to install
+* @returns {Wine} Wine object
+*/
+Wine.prototype.VK9 = function (vk9Version) {
+    print("NOTE: you need a driver that supports Vulkan enough to run VK9");
+    print("NOTE: wine version should be greater or equal to 3.5");
+    print("NOTE: work from 0.28.0");
+
+    if (typeof vk9Version === 'undefined')

```

Comment:
probably better: `typeof vk9Version === 'string'`

Reply:
I want that code to be executed if no variable is sent. If I do that, ```vk9version``` will never be taken into account.

Reply:
The problem is if someone passes a number or something else which is invalid. Therefore I think it would be better to test for string and use the default otherwise.